scripts/publish/src/getOtpTag.js

```diff
@@ -0,0 +1,14 @@
+// Builds opt tag for publish command.
+// Accepts both '--otp 123456' and '--otp=123456' format.
+module.exports = () => {
```

Comment:
I think using https://www.npmjs.com/package/yargs-parser could be better than implementing argument parsing here.